s3prl/downstream/speech_translation/config.yaml

```diff
@@ -0,0 +1,70 @@
+runner:
+  total_steps: 32000
+  gradient_clipping: 10
+  gradient_accumulate_steps: 8
+
+  log_step: 100
+  eval_step: 2000
+  save_step: 250
+  max_keep: 1
+  eval_dataloaders:
+    - dev
+    # - test
+
+optimizer:
+  name: Adam
+  lr: 2.0e-3
+
+# comment the whole scheduler config block
+# to disable learning rate scheduling
+scheduler:
+  name: sqrt_decay_schedule_with_warmup
+  num_warmup_steps: 10000
+
+downstream_expert:
+
+  src_lang: en
+  tgt_lang: de
+  post_process: sentencepiece
+  output_prefix: output-st # set prefix of output files
+
+  criterionrc:
+    criterion: label_smoothed_cross_entropy
+    label_smoothing: 0.1
+
+  taskrc:
+    data: data/covost_en_de ## set the data folder here
+    config_yaml: config.yaml
+    seed: 1
+    use_asr: False
+
+  asrrc:
+    weight: 0.3
+    vocab_file: spm-src_text.txt
+    bpe_tokenizer:
+      bpe: sentencepiece
+      sentencepiece_model: spm-src_text.model
+    datarc:
+      key: src_text # header in tsv
+
+
+  datarc:
+    train: train
+    dev: dev
+    test: test
+    max_tokens: 10000
+    num_workers: 4
+
+  # configuration of decoding
+  generatorrc:
+    beam: 5
+    max_len_a: 0
+    max_len_b: 400
+
+  # the model configuration and creation are delegated to fairseq
+  modelrc:
+    arch: s2t_transformer
```

Comment:
Maybe I miss something: where I could find the detailed settings of the model (e.g., dropout, layer size, etc.)

Reply:
As I mention in [README](https://github.com/s3prl/s3prl/blob/ce4a33d101ac8d4887a1539ca8146433a196d790/s3prl/downstream/speech_translation/README.md/#training), I have delegate the model configuration and creation to Fairseq and the detailed settings are defined in https://github.com/pytorch/fairseq/blob/741fd138c69eebc1fac09b590047da85cdeafe2d/fairseq/models/speech_to_text/s2t_transformer.py

For example, the setting of architecture `s2t_transformer` is 
https://github.com/pytorch/fairseq/blob/741fd138c69eebc1fac09b590047da85cdeafe2d/fairseq/models/speech_to_text/s2t_transformer.py#L406-L446

If there are some confusion about the setting of hyper parameters, I could add more explanation in config.

Reply:
Many thanks for the explanation. Then just for demonstration purposes, it would be great to add more explanation (probably also how to change that in the config)